iron-multi-selectable.html

```diff
@@ -108,6 +108,7 @@
         }
         // Check for items, since this array is populated only when attached
         if (this.fallbackSelection && this.items.length && !this._selection.get().length) {
+          this.fire('iron-fallback-selection-triggered', this.fallbackSelection);
```

Comment:
The reason it is not, is so that selectors do not have to have that specific item in their light dom. E.g. you do not need a `<section name="404">` in every single route selector.